testing/src/main/java/org/oppia/android/testing/profile/ProfileTestHelper.kt

```diff
@@ -99,6 +99,9 @@ class ProfileTestHelper @Inject constructor(
   /** Login to user profile. */
   fun loginToUser() = logIntoProfile(internalProfileId = 1)
 
+  /** Login to a new user profile, since profile #2 is not used as a default user anywhere in this helper. */
+  fun loginToNewUser() = logIntoProfile(internalProfileId = 2)
```

Comment:
how do we differentiate when using in the test when to use `loginToUser` and when to use `loginToNewUser `?

Reply:
We want to use `loginToNewUser` to test cases that there is no story progress at all, so it relies on profile 1 and using `loginToUser` as the default if we want to manipulate a user profile's progress. Comment added to clarify this